pkg/healthchecker/health_checker_windows.go

```diff
@@ -85,6 +85,13 @@ func getHealthCheckFunc(hco *options.HealthCheckerOptions) func() (bool, error)
 			}
 			return true, nil
 		}
+	case types.CsiProxyComponent:
+		return func() (bool, error) {
+			if _, err := powershell("Get-Process", types.CsiProxyComponent); err != nil {
```

Comment:
If we just check whether the process exists, wouldn't windows service already handled it?

Why do we need this health checker now then?

Reply:
I am currently looking into a way to connect to csi-proxy endpoint for NPD Health Checker to use to test if it is up and running. In the meantime, this was used to test it.